BSP/Simulator/Hardware/simulate.py

```diff
@@ -46,8 +46,13 @@ def update_CAN():
     """Periodically update the display state of the CAN bus"""
     global id_text, message_text
     can = CAN.read()
+    
+    if not can: # If can is empty list
+        can.extend(["Empty", "Empty"])
+
```

Comment:
Can you implement this logic in the `CAN.py` file instead? I want the `simulate.py` module to be as unaware as possible as to what's going on behind the scenes.